### PR TITLE
SEO-2024 Blazor ListView External Linking

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @ChristoIssac ,

Please review the changes for Blazor ListView External Linking.

Regards,
Yvone.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,14 +1,77 @@
-# Create Blazor ListView In Blazor Webassembly App
-A quick start Blazor WebAssembly app that allows you to configure the Blazor ListView component from Syncfusion with its available options. This project contains simple code customizations, as well as some important features like how to bind different types of data, group list items, add checkboxes to a list, and add nested list items.
+ # Create Blazor ListView In Blazor Webassembly App
```
Comment (Girija-Rajendran) [inline README.md:1]: make sure it is working properly

File: README.md [hunk 2]
```diff
@@ -1,14 +1,77 @@
...
-A quick start Blazor WebAssembly app that allows you to configure the Blazor ListView component from Syncfusion with its available options. This project contains simple code customizations, as well as some important features like how to bind different types of data, group list items, add checkboxes to a list, and add nested list items.
+ # Create Blazor ListView In Blazor Webassembly App
 
-Documentation: https://blazor.syncfusion.com/documentation/listview/getting-started/
-
-Online examples: https://blazor.syncfusion.com/demos/listview/default-functionalities
+The [Blazor ListView](https://www.syncfusion.com/blazor-components/blazor-listview?utm_source=github&utm_medium=listing&utm_campaign=blazor-lists-github-samples) is a list-like interface that allows you to select an item or multiple items. A quick start Blazor WebAssembly app that allows you to configure the Blazor ListView component from Syncfusion with its available options. This project contains simple code customizations, as well as some important features like how to bind different types of data, group list items, add checkboxes to a list, and add nested list items.
```
Comment (Girija-Rajendran) [inline README.md:3]: UTM param should be 

?utm_source=github&utm_medium=listing&utm_campaign=blazor-listview-github-samples

File: README.md [hunk 3]
```diff
@@ -1,14 +1,77 @@
...
+ # Create Blazor ListView In Blazor Webassembly App
 
-Documentation: https://blazor.syncfusion.com/documentation/listview/getting-started/
-
-Online examples: https://blazor.syncfusion.com/demos/listview/default-functionalities
+The [Blazor ListView](https://www.syncfusion.com/blazor-components/blazor-listview?utm_source=github&utm_medium=listing&utm_campaign=blazor-lists-github-samples) is a list-like interface that allows you to select an item or multiple items. A quick start Blazor WebAssembly app that allows you to configure the Blazor ListView component from Syncfusion with its available options. This project contains simple code customizations, as well as some important features like how to bind different types of data, group list items, add checkboxes to a list, and add nested list items.
 
 ## Project prerequisites
 
-Make sure that you have the compatible versions of Visual Studio 2019 and .NET Core SDK 3.1.3 in your machine before starting to work on this project.
```
Comment (Girija-Rajendran) [inline README.md:10]: don't make any changes here

File: README.md [hunk 4]
```diff
@@ -1,14 +1,77 @@
...
+## Sortable list
+Sort data programmatically in ascending or descending order. Also use custom sorting logic.
+
+## Group list
+Group ListView data programmatically. You can identify a group name from group headers in the view. Also use custom sorting logic.
+
+## Multilevel ListView
+The Blazor ListView represents a nested data object structure in a list of lists. The initial screen displays the first-level list items and next level will be displayed when interacting with it.
+
+### Selection
```
Comment (Girija-Rajendran) [inline README.md:27]: Why is this empty?

File: README.md (hunk 2 above)
Comment (Yvone-Atieno) [reply README.md:3]: resolved

File: README.md (hunk 4 above)
Comment (Yvone-Atieno) [reply README.md:27]: resolved

**Pull request merged**